examples/dbrVideoSmallTool/README.md

```diff
@@ -0,0 +1,35 @@
+```js
+/**
+ * function showDbrVideoSmallTool
+ * 
+ * Please feel free to modfiy `dbrVideoSmallTool.js` and `dbrVideoSmallTool.css` to match your scenario.
+ * 
+ * | parameter        | type           | description
+ * | ---------------- | -------------- | -----------
+ * | *(return value)* | HTMLDivElement | 
+ *     The element of the tool. You can put it somewhere else. By default it will be appended in body.
+ * | ---------------- | -------------- | -----------
+ * | callback         | function(txt)  | 
+ *     It is called each time a reliable result is obtained.
+ *     The same neigbour results would be regarded as one result.
+ * | ---------------- | -------------- | -----------
+ * | count            | number         | 
+ *     if `count` is 0, the window would not automatically close. Default 0.
+ *     if `count` == 1, close window after get a relibale result.
```

Comment:
close window after get a relibale result. => The application will close the window after getting a reliable result.